spec/classes/collectd_plugin_mysql_database_spec.rb

```diff
@@ -84,6 +84,13 @@
           facts.merge(collectd_version: '5.6')
         end
 
+        let :params do
```

Comment:
This is under the default options context, so you should not be specifying the params here. This block is to show what happens with the defaults, so what happens with host and port are undef?